pkg/db/db.go

```diff
@@ -1325,15 +1328,90 @@ func (h *DBHandler) DBSelectDeployment(ctx context.Context, tx *sql.Tx, appSelec
 		return nil, fmt.Errorf("deployments: row has error: %v\n", err)
 	}
 	return &Deployment{
-		EslVersion: row.EslVersion,
-		Created:    row.Created,
-		App:        row.App,
-		Env:        row.Env,
-		Version:    row.ReleaseVersion,
-		Metadata:   resultJson,
+		EslVersion:    row.EslVersion,
+		Created:       row.Created,
+		App:           row.App,
+		Env:           row.Env,
+		Version:       row.ReleaseVersion,
+		Metadata:      resultJson,
+		TransformerID: row.TransformerID,
 	}, nil
 }
 
+// Unused for now
+func (h *DBHandler) DBSelectDeploymentByTransformerID(ctx context.Context, tx *sql.Tx, transformerID TransformerID, limit uint) ([]Deployment, error) {
+	span, _ := tracer.StartSpanFromContext(ctx, "DBSelectDeployment")
```

Comment:
```suggestion
	span, ctx := tracer.StartSpanFromContext(ctx, "DBSelectDeploymentByTransformerID")
```
Context is used later, so we need to use the return value here.
And "DBSelectDeployment" is already used elsewhere as a span name.